designer/client/src/list/ListEdit.test.tsx

```diff
@@ -41,9 +41,56 @@ describe('ListEdit', () => {
     const $listCaption = screen.getByText('List items')
     const $listHint = screen.getByText('Enter a unique name for your list')
     const $listLink = screen.getByText('Add a new list item')
+    const $listDelete = screen.getByText('Delete')
```

Comment:
Think we're alright, but these old `getByText()` tests sometimes matched text not form elements

```suggestion
    const $buttonDelete = screen.getByRole('button', { name: 'Delete' })
```

Reply:
Resolved with 02835de15b88969cc9b12964dc5e0a45916d6d6b